diff -Naur mozc-2.23.2815.102+dfsg/src/unix/ibus/property_handler.cc mozc-2.23.2815.102+dfsg-patched/src/unix/ibus/property_handler.cc
--- mozc-2.23.2815.102+dfsg/src/unix/ibus/property_handler.cc	2018-04-03 08:03:55.000000000 +0900
+++ mozc-2.23.2815.102+dfsg-patched/src/unix/ibus/property_handler.cc	2021-01-30 11:16:11.917385884 +0900
@@ -80,7 +80,7 @@
 // On IBus 1.4.x, IBus expects that an IME should always be turned on and
 // IME on/off keys are handled by IBus itself rather than each IME.
 #if IBUS_CHECK_VERSION(1, 5, 0)
-const bool kActivatedOnLaunch = false;
+const bool kActivatedOnLaunch = true;
 #else
 const bool kActivatedOnLaunch = true;
 #endif  // IBus>=1.5.0
